src/openrct2/ride/Vehicle.h

```diff
@@ -453,6 +457,27 @@ enum : uint32_t
     VEHICLE_ENTRY_FLAG_DODGEM_CAR_PLACEMENT = 1u << 31,
 };
 
+constexpr const uint8_t ANIMATION_SPEED_MAX_EXPONENT = 32;
+constexpr const uint8_t STEAM_EFFECT_TRANSLATION_COEFFICIENT = 32;
+
+/*
+    First value is bitshift value where 31 is slowst and 0 is fastest
+    Second value is multiplier 0-7 (be warned a multiplier of 0 means animations will not play)
+    Exponent value in JSON objects is 32 - bitshift, such that "Exponent":12 translates to bitshift value of 20
+*/
+constexpr const uint8_t VehicleDefaultAnimationSpeeds[][2] = {
```

Comment:
Sorry that I forgot about this, but I would like you to create a struct with two `uint8_t`s, rather than this multidimensional array.